### DISCUSSION TITLE
Not Auto updating in Home Assistant

### DISCUSSION THREAD

**Discussion by AngusNB** [mrlt8/docker-wyze-bridge#1032] [Q&A] [answered]
I have Docker-Wyze-Bridge running in Home Assistant on a RPi3. Overall it works great.
I noticed that it hasn't updated to v2.4.0. It's still at v2.3.17. Auto update is on.
Can I manually update it? If so, how?

**Comment by mrlt8** [comment]
You may need to go into the add-ons for Home Assistant to check for updates?

**Reply by AngusNB** [reply]
I tried that. No updates are shown.

Also, the Documentation tab is showing "Addon 7094bb28_docker-wyze-bridge with version latest does not exist in the store"

**Reply by mrlt8** [reply]
Do you have the correct repo added?
```
https://github.com/mrlt8/docker-wyze-bridge
```

<img width="410" alt="repo" src="https://github.com/mrlt8/docker-wyze-bridge/assets/67088095/1d49ce7e-0bde-45e7-84b3-e55aedd9e6bc">

**Reply by AngusNB** [reply]
I do.

![image](https://github.com/mrlt8/docker-wyze-bridge/assets/34507363/0fcdbfc1-a76b-4db5-9b02-665dd252b52e)

I don't mind reinstalling if I can figure out how to save my config.

**Reply by AngusNB** [reply, marked as answer]
I did re-install. Just save the YAML config and pasted it to the new install.

Now I have both v2.3.17 and 2.4.0. I will uninstall the old one unless you want to try to determine what went wrong with the update.
I don't see any others having the issue so it may not be worth our time.
Thanks for getting back to me. I'm looking forward to the new features in 2.4.0.

**Reply by superdong69** [reply]
This shouldn't be the answer. Autoupdating in Home Assistant DOES NOT work. Neither is there a way to update the addon. Please fix.

**Reply by mrlt8** [reply]
Thanks for the feedback. I believe this is probably because we moved and separated the config for HA into its own directory.

Will look into this.

**Reply by AngusNB** [reply]
Updates worked fine from 2.4.0 to 2.6.0.
I don't think the recent issue I had with 2.7.0 is the same issue I had with 2.3.x. although the WebUI version doesn't match the app version. I still don't know if that's intentional or not.

**Reply by superdong69** [reply]
Yeah, it HAD been working just fine, as I've never had to update myself before. But I've been having a ton of issues where the streams just stop and I can't see them in Frigate anymore.
